### ISSUE TITLE
DOC: Typo on numpy/numpy/lib/npyio.py?

### ISSUE THREAD

**Issue by Scoodood** [numpy/numpy#10041]
numpy/numpy/lib/npyio.py
```
Line 628:  numpy.savez : Save several arrays into an uncompressed ``.npz`` file format
```
compressed or uncompressed?

Other line suggest *.npz is a compressed archive

**Comment by kchomski-reef** [comment]
I think this line should be added to the function docstring to make things clear :wink:
```python
savez_compressed : Save several arrays into a compressed ``.npz`` archive
```
Both `savez` and `savez_compressed` call `_savez` inside:
```python
def savez(file, *args, **kwds):
    _savez(file, args, kwds, False)

def savez_compressed(file, *args, **kwds):
    _savez(file, args, kwds, True)
```

which is defined like this:
```python
def _savez(file, args, kwds, compress, allow_pickle=True, pickle_kwargs=None)
```

**Comment by Ayush-A97** [comment]
#10133 , fixed the typo.

**Comment by eric-wieser** [comment]
@kchomski-reef: Want to make a patch to add that line?

**Comment by mattip** [comment]
fixed by #10113
